src/foam/u2/wizard/tests.jrl

```diff
@@ -0,0 +1,103 @@
+p({
+  class:"foam.nanos.test.Test",
+  id:"DaoWAOTest",
+  description: "Unit test for DAO Wizardlet Access Object",
+  language: 0,
+  code:"""
+    async function execute(x) {
```

Comment:
I believe there's a way to do this in a js file but still register it in a journal